### DISCUSSION TITLE
Demo source code

### DISCUSSION THREAD

**Discussion by grndctrl** [payloadcms/payload#523] [Q&A] [answered]
Hi,

Is the source code for the demo at https://demo.payloadcms.com available to browse through?

I'm specifically interested in the SEO field.

Thanks!

**Comment by DanRibbens** [comment, marked as answer]
We are planning to make the demo repo public after a little code cleanup.

You can use this SEO plugin https://github.com/payloadcms/payload-plugin-seo. I believe the demo SEO fields are based on this.

Let us know if you need anything else.

**Comment by grndctrl** [comment]
Awesome! Looking forward to it.
